src/util/dev/builder.ts

```diff
@@ -369,7 +370,8 @@ export async function getBuildMatches(
   nowConfig: NowConfig,
   cwd: string,
   yarnDir: string,
-  output: Output
+  output: Output,
+  fileList: string[]
```

Comment:
Where is `getBuildMatches()` called? Do you need to add the additional parameter at each call site?

Reply:
@styfle it is called here https://github.com/zeit/now-cli/pull/2591/files#diff-81b0317d818fc86303285d3ed31c8dd6R300, with the new parameter